app/views/plants/show.html.erb

```diff
@@ -40,7 +40,7 @@
               </li>
             <% end %>
           </ul>
-          <nav class="mt-4 pagy-container">
+          <nav class="mt-4 pagy">
             <%== pagy_nav(@pagy) %>
           </nav>
```

Comment:
Right now it generates this html and it is a bit weird to have `nav` inside `nav` and with the same class:
<img width="449" alt="Screenshot 2024-09-05 at 19 46 54" src="https://github.com/user-attachments/assets/ae2b08e7-bca0-43f6-97e4-9f9ee077b4a2">

Looks like `pagy_nav` already generates a nav element with class `pagy` so we can change this to something like this :) 
```suggestion
          <div class="mt-4">
            <%= pagy_nav(@pagy) %>
          </nav>
```